lib/src/view/study/study_list_screen.dart

```diff
@@ -123,7 +138,11 @@ class _StudyListScreenState extends ConsumerState<StudyListScreen> {
 
     return PlatformScaffold(
       appBar: PlatformAppBar(
-        title: Text(sessionUser != null ? context.l10n.studyMenu : context.l10n.studyAllStudies),
+        title: Text(
```

Comment:
You should use the `AppBarTitleText` widget here, as it will auto size the text to fit in.